tests/grammar/vyper.lark

```diff
@@ -74,11 +74,12 @@ event_def: _EVENT_DECL NAME ":" ( event_body | _PASS )
 
 // Types
 array_def: (NAME | array_def) "[" (DEC_NUMBER | NAME) "]"
-tuple_def: "(" ( NAME | array_def | tuple_def ) ( "," ( NAME | array_def | tuple_def ) )* [","] ")"
+dyn_array_def: "DynArray" "[" (NAME | array_def | dyn_array_def) "," (DEC_NUMBER | NAME) "]"
+tuple_def: "(" ( NAME | array_def | dyn_array_def | tuple_def ) ( "," ( NAME | array_def | dyn_array_def | tuple_def ) )* [","] ")"
```

Comment:
Tuple of `DynArray`s shouldn't cause any issues right?

Reply:
Theoretically no, we should add it to list of things to QA